jujugui/static/gui/src/app/components/budget-chart/budget-chart.js

```diff
@@ -4,6 +4,8 @@
 const PropTypes = require('prop-types');
 const React = require('react');
 
+require('./_budget-chart.scss');
```

Comment:
I don't think we have any duplicate requires, but are they deduped as well?

Reply:
They are deduped in the browserify scss builder! But I think you're right that we don't include anything twice anyway.